_data/tutorials.yaml

```
@@ -712,3 +712,22 @@ error-handling:
     scala: disabled
     swift: disabled
     confluent: enabled
+
+schedule-ktable-ttl-aggregate:
+  title: "How to implement TTL-based cleanup to expire data in a KTable created via aggregate"
+  meta-description: "How to implement TTLs to expire data in the middle of a topology"
```

```suggestion
  meta-description: "implement TTLs to expire data in the middle of a topology"
```

(see https://github.com/confluentinc/kafka-tutorials/pull/918 for details)